samples/bootloader/src/main.c

```diff
@@ -130,6 +134,8 @@ static void boot_from(u32_t *address)
 		nvic->ICPR[i] = 0xFFFFFFFF;
 	}
 
+	printk("Booting.\r\n");
```

Comment:
This should include the address being booted. Don't we have that print statement somewhere else?

Reply:
We have

	printk("Attempting to boot from address 0x%x.\n\r", (u32_t)address);

at the beginning of `boot_from`. I can add the address here as well.